values/jobs/gitea-prepare.gotmpl

```diff
@@ -1,6 +1,7 @@
 {{- $v := .Values }}
 {{- $c := $v.apps }}
 {{- $g := $c.gitea }}
+{{- $otomiValues := pick $v "cluster" "otomi" "teamConfig" }}
```

Comment:
I am not sure if passing the whole teamConfig is a good idea, as it can grow significantly.
Does the task need only team names?

Reply:
no, selfservice flags...data is super small

Reply:
But you are passing the whole team including services